### DISCUSSION TITLE
Reopen in dev container context menu

### DISCUSSION THREAD

**Discussion by windmillcode0** [devcontainers/community#106] [Ideas]
Greetings  I was wondering if there is a feature to right click on a folder in explorer view and when the context menu opens you can choose "Reopen in Dev Container" this saves from the tedious tasks of opening that folder in a new vscode window, waiting for the extension to load then getting to the command prompt

Also a feature of open in new window in case you want to keep the original window open

**Comment by samruddhikhandale** [comment]
Hi 👋 

Thanks for sharing your thoughts, I don't believe we have an option today. However, the `devcontainers` team does not maintain the VS Code UX or the `Dev containers` extension.

Can you open an issue in https://github.com/microsoft/vscode-remote-release/issues? The VS Code team should be able to help you provide guidance. Thanks!
